src/masonite/container/container.py

```diff
@@ -211,21 +211,16 @@ def resolve(self, obj, *resolving_arguments):
                         "This container is not set to resolve parameters. You can set this in the container"
                         " constructor using the 'resolve_parameters=True' keyword argument."
                     )
-        try:
-            if self.remember:
-                if not inspect.ismethod(obj):
-                    self._remembered[obj] = objects
-                else:
-                    signature = "{}.{}.{}".format(
-                        obj.__module__, obj.__self__.__class__.__name__, obj.__name__
-                    )
-                    self._remembered[signature] = objects
-            return obj(*objects)
-        except (TypeError,) as e:
-            exception = ContainerError
-            exception.from_obj = obj
 
-            raise exception(str(e) + " while calling {}".format(obj)) from e
```

Comment:
@girardinsamuel removing this improves the stack trace. We were catching specific exceptions and then reraising a new exception. So anytime you application actually threw a typeError, you would never know where it came from because we were re-raising a new error. 

The weird part is I'm not sure why I ever did this. I don't see a need to do this ...

Reply:
That's a nice fix because it was sometimes complex to find out what and where was the error raised !